composer.json

```diff
@@ -34,8 +34,8 @@
     "require":           {
         "php":                               ">=5.3.23",
         "doctrine/doctrine-module":          "0.9.*",
-        "doctrine/orm":                      ">=2.4,<2.6-dev",
-        "doctrine/dbal":                     ">=2.4,<2.6-dev",
+        "doctrine/orm":                      ">=2.5,<2.7-dev",
+        "doctrine/dbal":                     ">=2.5,<2.7-dev",
```

Comment:
The goal of adding a version check is to preserve the compatibility with 2.4 too